cmd/podman/machine/machine.go

```diff
@@ -24,6 +24,10 @@ var (
 		PersistentPreRunE:  noOp,
 		PersistentPostRunE: noOp,
 		RunE:               validate.SubCommandExists,
+		Annotations: map[string]string{
+			registry.UnshareNSRequired: "",
+			registry.ParentNSRequired:  "",
+		},
```

Comment:
I don't think this is correct.

Reply:
this is what jhonce had me do.

Reply:
@baude Not sure what context we were talking about when I made that recommendation. @Luap99 is correct `UnshareNSRequired` will have the CLI enforce this command can only be run after `unshare`. `ParentNSRequired` implies root-required.